openfl/experimental/runtime/runtime.py

```diff
@@ -17,7 +17,6 @@ def __init__(self):
 
         This serves as a base interface for runtimes that can run FLSpec flows.
         """
-        pass
```

Comment:
For all changes with `pass` removed: it is more readable when it is explicit, even if python considers this to be syntactically correct.

Reply:
Agree

Reply:
resolved in commits 339f74f5e7306526eceb2bdcb7326ed724de1f29 and d3733e3f9883f0c7f436c6ad5e4f0aec3b7bb862